feat(core): added template for issues, feature requests and prs (#364)
fetchai/uAgents: .github/ISSUE_TEMPLATE/feature-request.md

name: Feature request
description: Suggest a new feature for uAgents

> title: "[Feature request]: <title>"

Reviewer: ```suggestion
title: "<title>"
```

labels: ["enhancement", "unconfirmed"]
body:
  - type: markdown
    attributes:
      value: |

>         Thank you for suggesting a new feature. Please fill out the form to ensure we have everything to get things started.

Reviewer: ```suggestion
        Thank you for suggesting a new feature. Please fill out the form to ensure we have everything to get started.
```

>         Thank you for suggesting a new feature for uAgents. Please provide the following details to ensure we have all the details to get things started.

Reviewer: ```suggestion
        Thank you for suggesting a new feature. Please fill out the form to ensure we have everything to get things started.
```

  - type: checkboxes
    id: prerequisites
    attributes:
      label: Prerequisites

>       description: Please confirm before submitting any new feature request.

Reviewer: ```suggestion
      description: Please confirm before submitting any new feature request:
```

>       description: Please confirm before submitting any new feature request.

Reviewer: ```suggestion
      description: Please confirm before submitting any new feature request:
```
